src/py/flwr/cli/run/run.py

```diff
@@ -70,11 +70,18 @@ def run(
     if engine == Engine.SIMULATION:
         num_supernodes = config["flower"]["engine"]["simulation"]["supernode"]["num"]
 
+        backend_config = None
+        if "backend_config" in config["flower"]["engine"]["simulation"]:
+            backend_config = config["flower"]["engine"]["simulation"]["backend_config"][
+                "config"
+            ]
+
```

Comment:
Is this equivalent? Assuming that we want to pass everything under `[flower.engine.simulation.backend_config]` to the `_run_simulation` call.

```
backend_config = config["flower"]["engine"]["simulation"].get("backend_config", None)
```

Reply:
Yes, this way is cleaner